hlog/hlog.go

```diff
@@ -1,3 +1,5 @@
+// +build !enable_binary_log
```

Comment:
Why?

Reply:
I made a judgement call that http logger will not want binary encoding - just like how syslog and binary log doesn't go together. Do you think otherwise ?

Reply:
HTTP handlers are just tools to add context to the logs. The format of the output does not matter.

Reply:
my bad. i didn't read hlog in detail... Will read thru and make necessary changes in a few days..

Reply:
Enabled hlog_test.go for binary test too. hlog_example_test.go - i left it just text.